Python/symtable.c

```diff
@@ -2650,9 +2712,6 @@ symtable_visit_argannotations(struct symtable *st, asdl_arg_seq *args)
 {
     Py_ssize_t i;
 
-    if (!args)
```

Comment:
Maybe keep the check as an assertion? assert(args != NULL)? Same remark in symtable_visit_params().